_includes/research-platform/projects-collapsable.html

```diff
@@ -0,0 +1,35 @@
+<section id="projects" class="container content-section">
+    <div class="row">
+        <div class="col-lg-12">
+            <p>
+                We aim to connect students and research leaders in ML/AI 
+                by helping the students discover projects suited to their interests
+                and research labs showcase their work and available opportunities
```

Comment:
I think "Allowing research labs..." would sound better

Reply:
Let's discuss it on the next meeting!